src/lti/message_validators/submission_review_message_validator.php

```diff
@@ -8,19 +8,19 @@ public function can_validate($jwt_body) {
 
     public function validate($jwt_body) {
         if (empty($jwt_body['sub'])) {
-            throw new LTI_Exception('Must have a user (sub)');
+            throw new LTI_Message_Validation_Exception('Must have a user (sub)');

```

Comment:
I was somewhat perplexed to see that, while this import is being made from a different directory, it shares the same namespace as its parent. I'm not changing that here, and don't know if that's right or wrong.